plugins/modules/zos_data_set.py

```diff
@@ -467,6 +479,18 @@
         type: bool
         required: false
         default: false
+      force:
+        description:
+          - Specifies that the data set can be shared with others during a member
```

Comment:
I think this doc came from zos_blockinfile which is slightly different in its use with `force` where that force is applied to the data set in its entirety for read and write. Where here ware only supporting member deletes as DISP=SHR.

So maybe alter this:
```
      - This is helpful when a data set is being used in a long running process
        such as a started task and you are wanting to update or read.
```
to 
```
      - This is helpful when a data set is being used in a long running process
        such as a started task and you delete a member. 
```


I did fix the zos_blockfile doc a day ago, this should read:

      - The I(force=true) option enables sharing of data sets through the disposition
         I(DISP=SHR).
      - The I(force=true) only applies to data set members when I(state=absent) and
         I(type=MEMBER).

I added an extra line to highlight even more this support is just for data set members

Reply:
how embarrassing of me, you're right, i did lift it from that doc but i thought i'd modified it

Reply:
i wonder if there's a path to using document fragments here since we have 2 of the same docs except one is nested under "batch"

Reply:
There is a path to use doc fragements, we can do this , and have been wanting to for idk, couple of yrs now :) , I share more on this in one of our meetings.